tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -720,7 +709,7 @@ func init() {
     actionCreateCmd.Flags().BoolVar(&flags.action.copy, "copy", false, "treat artifact as the name of an existing action")
     actionCreateCmd.Flags().BoolVar(&flags.action.sequence, "sequence", false, "treat artifact as comma separated sequence of actions to invoke")
     actionCreateCmd.Flags().StringVar(&flags.action.kind, "kind", "", "the kind of the action runtime (example: swift:3)")
-    actionCreateCmd.Flags().StringVar(&flags.action.shared, "shared", "", "shared action (default: private)")
+    actionCreateCmd.Flags().BoolVar(&flags.common.shared, "shared", false, "make an action publically accessible")
```

Comment:
maybe also add what the default value is.. something like `(default: action is private)`.. maybe shorter `(default: private)`